src/runtime/audience-activity-listener-test.js

```diff
@@ -94,8 +94,9 @@ describes.realWin('AudienceActivityEventListener', {}, (env) => {
     );
 
     const path = new URL(capturedUrl);
-    expect(path.pathname).to.equal(
-      '/swg/_/api/v1/publication/pub1/audienceactivity'
+    expect(path.toString()).to.equal(
+      '$frontend$/swg/_/api/v1/publication/pub1/audienceactivity?sut=' +
```

Comment:
What about checking the encoded value here to make sure it fixes the specific issue? (i.e., no white space in the url)
`'...?sut=ab%2Bc'`